pedestrian-generator/poetry.lock

```diff
@@ -0,0 +1,424 @@
+[[package]]
+name = "appdirs"
```

Comment:
should we really check in the lock files?

Reply:
The lockfile is there to ensures that everyone has exactly the same version of all dependencies. See e.g. https://python-poetry.org/docs/basic-usage/#commit-your-poetrylock-file-to-version-control.